lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const worldStateData = require('warframe-worldstate-data');
+
+/**
+ * Translator to convert in-game names to their localizations
+ */
+class Translator {
+		
+	constructor(lang = 'en_US') {
+		this.data = worldStateData[lang];
+		if(typeof this.data !== "object") {
```

Comment:
Actually `warframe-worldstate-data` exports all datas into "en_US" object. Defaulting to "en_US" allow to be retrocompatible with warframe-worldstate-data structure. The line 12 check if a translation object exists, otherwise throws a error. So `this.data` cannot be used after.